### DISCUSSION TITLE
How to compile for microbit v2 based hardware prototype

### DISCUSSION THREAD

**Discussion by CaptainCredible** [microsoft/pxt-arcade-hardware-designs#3] [Q&A] [answered]
Can i compile a makecode arcade sketch for the microbit v2 to test it with my prototype hardware?

**Comment by mmoskal** [comment, marked as answer]
Yes! it's described here https://github.com/microsoft/pxt-arcade-hardware-designs/tree/master/microbit-shield#testing-basic-operation
